readme.md

```diff
@@ -61,12 +61,11 @@ flatpak install com.usebruno.Bruno
 
 # On Linux via Apt
 sudo mkdir -p /etc/apt/keyrings
+sudo apt-get update && sudo apt-get install gpg
```

Comment:
I believe `apt` should be used for end-users and `apt` for scripts.

[Source](https://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get)